nerfstudio/data/datamanagers/full_images_datamanager.py

```diff
@@ -68,6 +69,15 @@ class FullImageDatamanagerConfig(DataManagerConfig):
     """The image type returned from manager, caching images in uint8 saves memory"""
     max_thread_workers: Optional[int] = None
     """The maximum number of threads to use for caching images. If None, uses all available threads."""
+    train_cameras_sampling_strategy: Literal["random", "fps"] = "random"
+    """Specifies which sampling strategy is used to generate train cameras, 'random' means sampling 
+    uniformly random without replacement, 'fps' means farthest point sampling which is helpful to reduce the artifacts 
+    due to oversampling subsets of cameras that are very closer to each other."""
+    train_cameras_sampling_seed: int = 42
+    """Random seed for sampling train cameras. Fixing seed may help reduce variance of trained models across 
+    different runs."""
+    fps_reset_every: int = 100
+    """The number of iterations before one resets fps sampler repeatly"""
```

Comment:
@jb-ye, this option is great, but I think the docstring should be improved, it took me some time to understand the logic here.  I have a small doubt, if `fps_reset_every < num_train_cameras` then is it possible to construct a setting under which it is possible to never sample a training camera(s) during training using this method, taking into account the weighting of the `train_unsampled_epoch_count` cameras in L:161?

Reply:
I haven't tested your hypothetic setting. In theory, when fps_reset_every is very small, the sampling would be more similar to random with replacement except that the effect of using `train_unsampled_epoch_count` we can at least bound its maximum though the use of weighting.